README.md

```diff
@@ -12,16 +12,23 @@ All Markdown files here are run through [tianon's fork of `markdownfmt`](https:/
 -	create a `README-short.txt` (required, 100 char max)
 -	create a `content.md` (required)
 -	create a `license.md` (required)
+-	create a `github-repo` (required)
```

Comment:
I think we should probably either add `.txt` on this file, or remove `.txt` on `README-short` :cry: